server/src/main/java/org/opensearch/cluster/metadata/AutoExpandReplicas.java

```diff
@@ -69,7 +69,7 @@ public final class AutoExpandReplicas {
         Property.IndexScope
     );
 
-    private static AutoExpandReplicas parse(String value) {
```

Comment:
we can directly use `AutoExpandReplicas autoExpandReplicas = AutoExpandReplicas.SETTING.get(settings);` wherever we need it instead.